docs/user-guide/gantt-chart/README.md

```diff
@@ -28,17 +29,37 @@ The **Gantt chart** in OpenProject displays the work packages in a timeline. You
 
 ## Activate the Gantt chart
 
-A Gantt chart can be activated in any work package table, to display the work packages in a timeline view.
+A Gantt chart view can be activated either for a specific project, or on the project overarching level if you need to view the timeline for multiple projects. 
```

Comment:
"cross-project" (vs "overarching) here too perhaps?

Reply:
we seem to have used overarching in the documentation before. I will to try to keep a mix of both, so that a user has a better chance of a successful search in the docs regardless of the keyword choice